scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -1,6 +1,7 @@
 # IPC gullwing generator
 
-This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages from the dimensions of the package. (The suggested footprint is only used to determine the size of the exposed pad.) Fillet size definitions can be changed by pointing to a personalized IPC parameter file. The script generates footprints in zero orientation version A (pin 1 at top left corner).
+This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages from the dimensions of the package. (The suggested footprint is generally ignored. It is only used to define the exposed pad if necessary to fulfil thermal or EMC requirements defined in the datasheet.)
```

Comment:
Typo of 'fulfill' (missing the last 'l').

Reply:
Well fulfil is the UK spelling